custom/enikshay/user_setup.py

```diff
@@ -29,125 +21,6 @@
     PRIVATE_SECTOR_WORKER_ROLE,
 )
 from .models import AgencyIdCounter, IssuerId
-from six.moves import range
-from six.moves import map
-
-TYPES_WITH_REQUIRED_NIKSHAY_CODES = ['sto', 'dto', 'tu', 'dmc', 'phi']
-LOC_TYPES_TO_USER_TYPES = {
-    'phi': ['to', 'tbhv', 'mo-phi'],
-    'tu': ['sts', 'stls'],
-    'dmc': ['lt-dmc'],
-    'cdst': ['lt-cdst'],
-    'dto': ['dto', 'deo'],
-    'cto': ['cto'],
-    'sto': ['sto'],
-    'drtb-hiv': ['drtb-hiv'],
-}
-
-
-def skip_custom_setup(domain, request_user):
-    return True  # We're gonna revisit this, turning off for now
-    return not toggles.ENIKSHAY.enabled(domain) or request_user.is_domain_admin(domain)
-
-
-def clean_user_callback(sender, domain, request_user, user, forms, **kwargs):
-    if skip_custom_setup(domain, request_user):
-        return
```

Comment:
Given that `skip_custom_setup` returns `True`, this is never called, nor is anything only used inside it.